tests/python/frontend/keras/test_forward.py

```diff
@@ -209,6 +211,15 @@ def test_forward_activations(self, keras_mod):
             keras_model = keras_mod.models.Model(data, x)
             verify_keras_frontend(keras_model)
             verify_keras_frontend(keras_model, need_transpose=False, layout="NHWC")
+        # test invalid attribute alpha=None for LeakyReLU
+        if package_version.parse(keras_mod.__version__.split("-tf")[0]) <= package_version.parse(
+            "2.3.1"
+        ):
```

Comment:
Why do you need such check on the package version?

Reply:
The following model is generated using keras (version 2.2) with the script. However, this model is not a valid model, which should be rejected in TVM front-end checking.  
```
input_shape = (2, 3, 4)
input_data = np.random.random(input_shape)
x = layers.Input(shape=input_shape[1:], dtype='float32')
layer = keras.layers.LeakyReLU(alpha=None)  
layer.set_weights(layer.get_weights())
y = layer(x)
model = models.Model(x, y)
```

![image](https://user-images.githubusercontent.com/29506758/233930829-dcc3774b-c81d-4b3a-9a65-625bbaf87980.png)

However, in the latest keras. Keras found `keras.layers.LeakyReLU(alpha=None)` is not a valid usage and added an exception value checking to avoid generating the invalid model ([see this issue](https://github.com/tensorflow/tensorflow/pull/47017)). Thus, for triggering this crash, we should use keras <=2.3.1.

Reply:
I see your point, thank you. I'm not sure if it is the best option to have such condition in the test. Another variant, we can catch exception from TF if keras version is higher as 2.3.1 and if no exception was generated by TF then we will try to verify our frontend.
Current option is also LGTM. Let's discuss with other folks what is the best option in the test. @AndrewZhaoLuo, @leandron what do you think, should we check Keras version in test?

Reply:
I think we can do that, as we are not very restrictive in terms of which Keras version we support. Some version validation is done so that we can update dependencies and keep CI running, but at some point, tests that only pass with ancient versions should be removed. That's the only case I see it is OK to add version checking within a test.

Reply:
And the reasonable question: how we can track such tests, which can be removed when the version starts to be ancient? Should be a github issue created or specific comment in the code can be used?

Reply:
I personally think a comment will suffice for this. 

Github issues should be more active problems imo. Having one hang out for a long time is bad (though I am guilty of this :P).